docs/dwarf-op.rst

```diff
@@ -119,13 +120,13 @@ Command options
 ``--optimize``
     Performs a job search for unoptimized dwarves. Run
     ``dwarf-op --list job_distribution`` to see how jobs are distributed.
-``--applyjobs "[" <job> [<job> ...] "]"``
+``--applyjobs "<job>[,<job>,...]"``
     Applies the listed jobs to the selected dwarves. Run
     ``dwarf-op --list jobs`` to see available jobs.
-``--applyprofessions "[" <profession> [<profession> ...] "]"``
+``--applyprofessions "<profession>[,<profession>,...]"``
     Applies the listed professions to the selected dwarves. Run
     ``dwarf-op --list professions`` to see available professions.
-``--applytypes "[" <profession> [<profession> ...] "]"``
+``--applytypes "<profession>[,<profession>,...]"``
```

Comment:
```suggestion
``--applytypes <profession>[,<profession> ...]``
```

Reply:
I wrestled with whether to omit the last comma but then thought about what I've been taught to do in math which is `1,2,3,...,N`

So I figured using that well known listing methodology would be best for people to understand what to do. Though they probably won't be dumb enough to try `jobs,Miner, Admin, Farmer`. In any case, I'll drop one of the things, but not both (just in case they are). Do you want the quotes, or the final comma?

Reply:
Reference syntax docs don't use the final comma, e.g.
![image](https://user-images.githubusercontent.com/977482/186340588-1280db45-9cac-4b3d-8f70-454161275580.png)
(that's from `man apropos`) and that's the standard I've been applying to other DFHack docs. Syntax docs also don't add quotes in -- the user adds them if they're needed, just like any single param that contains spaces. Let's go with no final comma and no quotes.

Reply:
Ok. An established convention for command line is enough. More applicable than math at any rate. Though I note your suggestions put a space, I think I'll omitt that to be in line with - what despite being a single data point - apropos which I am going to assume is representative of all similar commands.